miden-lib/src/lib.rs

```diff
@@ -5,7 +5,19 @@ extern crate alloc;
 
 use assembly::{utils::Deserializable, Library, LibraryNamespace, MaslLibrary, Version};
 
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "testing")]
+pub mod common;
```

Comment:
Since we use this for regular tests as well, should we add `cfg(test)` here too? i.e., use `#[cfg(any(test, feature = "testing"))]`.

Reply:
changed